zerver/lib/actions.py

```diff
@@ -825,6 +882,7 @@ def check_stream_name(stream_name):
         raise JsonableError("Invalid stream name")
 
 def send_pm_if_empty_stream(sender, stream, stream_name):
+    # type: (UserProfile, Optional[Stream], text_type) -> None
```

Comment:
Well I didn't exactly see what the function was doing, I saw that there was a `stream is None` somewhere in the body, so I annotated it as `Optional[Stream]`. Looking at the function, it does seem that this code is somewhat wrong, since it doesn't make sense to check for `stream.realm != sender.realm` before checking `stream is None`.

I think I should annotate it as `Stream` for now, since passing `None` at runtime will give an `AttributeError`.